src/components/Election/ElectionResults.tsx

```diff
@@ -0,0 +1,117 @@
+import { Tdata } from "../../types";
+
+import { FormattedMessage } from "react-intl";
+
+export function ElectionResults({ data }: { data: Tdata }) {
+  const {
+    totalWorkers,
+    worksCouncilSize,
+    minorityGender,
+    workplaceGenderQuota,
+    totalCandidates,
+    notEnoughSeats,
+    moreVotesThanWorkers,
+    suggestMoreSeats,
+    suggestedSeats,
+    totalVotes,
+  } = data;
+
+  const minorityGenderHasMembers = data[`num${minorityGender}`] > 0;
+
+  const numMinorityWorkers = workplaceGenderQuota[minorityGender];
+  const isQuotaDisabled =
+    worksCouncilSize > 1 && minorityGenderHasMembers && numMinorityWorkers;
+
+  return (
+    <div>
+      <h2>Election Stats</h2>
+      <div className="input-control">
+        <label htmlFor="totalWorkers">
+          <FormattedMessage
+            id="label.workerCount"
+            defaultMessage={"Worker Count"}
+          />
+        </label>
+        <span className="cell" id="totalWorkers">
+          {totalWorkers}
+        </span>
+      </div>
+      <div className="input-control">
+        <label htmlFor="numSeats">
+          <FormattedMessage
+            id="label.worksCouncilSeatCount"
+            defaultMessage={"Works Council Seats"}
+          />
+        </label>
+        <span className="cell" id="numSeats">
+          {worksCouncilSize}
+        </span>
+      </div>
+      {/* TODO: how does minority gender work with single member works councils? */}
+
+      <div className="input-control">
+        <label htmlFor="workplaceGenderQuota">
+          <a href="https://en.wikipedia.org/wiki/D%27Hondt_method">D'Hondt</a>
+          &nbsp;Gender Quota
+        </label>
+      </div>
+      <div
+        className={
+          !data.isGenderQuotaAchieved && isQuotaDisabled ? "error" : "success"
+        }
+        id="workplaceGenerQuota"
+      >
+        {isQuotaDisabled ? (
+          <>
+            {/* todo: replace with ICU pluralisation */}
+            {`There ${
+              data.isGenderQuotaAchieved
+                ? numMinorityWorkers > 1
+                  ? "are"
+                  : "is"
+                : "should be"
+            } at least ${numMinorityWorkers} works council member${
+              numMinorityWorkers > 1 ? "s" : ""
+            } for the minority gender (${minorityGender})
+          `}
+          </>
+        ) : (
+          `Minority gender does not apply ${
+            worksCouncilSize < 5
+              ? "when the works council has less than 5 seats"
+              : "when the minority proportion is so small that they aren't allocated any seats"
```

Comment:
A missing third case is when the candidates themselves are just exclusively composed of majority gender. Some people might thing that the election/results are invalid. They are still legally valid.

Reply:
ah, thank you!

Reply:
@shushugah in this case in the UI, no message is shown